Index: server/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.idea\n.ipynb_checkpoints\n.mypy_cache\n.vscode\npycache\n.pytest_cache\nhtmlcov\ndist\nsite\n.coverage\ncoverage.xml\n.netlify\ntest.db\nlog.txt\nPipfile.lock\nenv3.*\nenv\ndocs_build\nvenv\ndocs.zip\narchive.zip\n\n# vim temporary files\n~\n..sw?\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/.gitignore b/server/.gitignore
--- a/server/.gitignore	(revision c9237e932e7217bc58d65552b37d516bf115990c)
+++ b/server/.gitignore	(date 1679926894038)
@@ -14,7 +14,7 @@
 log.txt
 Pipfile.lock
 env3.*
-env
+.env
 docs_build
 venv
 docs.zip
Index: server/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/.env b/server/.env
new file mode 100644
--- /dev/null	(date 1679926907359)
+++ b/server/.env	(date 1679926907359)
@@ -0,0 +1,6 @@
+
+host = localhost
+port = 8889
+user = root
+password = root
+database = Bezebet
\ No newline at end of file
